src/libLookup/Lookup.cpp

```diff
@@ -2049,10 +2046,8 @@ void Lookup::CommitTxBlocks(const vector<TxBlock>& txBlocks) {
               "New lookup node - Already should have latest state by now.");
     if (GetDSInfo()) {
       if (!m_currDSExpired) {
-        if (FinishNewJoinAsLookup()) {
```

Comment:
is `FinishNewJoinAsLookup` still required
?

Reply:
Not required anymore. Fixed